lms/envs/common.py

```diff
@@ -408,6 +408,9 @@
 
     # Enable LTI Provider feature.
     'ENABLE_LTI_PROVIDER': False,
+
+    # Enable self-paced courses.
+    'ENABLE_SELF_PACED_COURSES': False,
```

Comment:
Please use a `ConfigurationModel` and the `enabled` property, as described here: https://openedx.atlassian.net/wiki/display/PLAT/Application+Configuration+and+Feature+Flagging